content/help/glossary/tls-origination.md

```diff
@@ -3,7 +3,7 @@ title: TLS Origination
 ---
 
 TLS origination occurs when an Istio proxy (sidecar or egress gateway) is configured to accept unencrypted
-internal HTTP connections, and then encrypt the request before forwarding it to either a simple or mutual TLS
-secured HTTPS server. This is the opposite of [TLS termination](https://en.wikipedia.org/wiki/TLS_termination_proxy)
+internal HTTP connections, encrypt the requests, and then forward them to either simple or mutual TLS
```

Comment:
suggest rephrasing /breaking up the sentence.. 
and then forward them to a HTTPS server thats secured using simple or mutual TLS.